etc/profile-a-l/electron-mail.profile

```diff
@@ -1,55 +1,39 @@
 # Firejail profile for electron-mail
-# Description: Unofficial desktop app for several E2E encrypted email providers
+# Description: Unofficial desktop app for the Proton Mail E2E encrypted email provider
 # This file is overwritten after every install/update
 # Persistent local customizations
 include electron-mail.local
 # Persistent global definitions
 include globals.local
 
+# breaks tray functionality
+ignore dbus-user none
+ignore disable-mnt
+
 noblacklist ${HOME}/.config/electron-mail
 
-include disable-common.inc
-include disable-devel.inc
-include disable-exec.inc
-include disable-interpreters.inc
-include disable-programs.inc
-include disable-shell.inc
-include disable-xdg.inc
+# sh is needed to allow Firefox to open links
+#include disable-shell.inc
 
 mkdir ${HOME}/.config/electron-mail
 whitelist ${HOME}/.config/electron-mail
-whitelist ${DOWNLOADS}
 
-include whitelist-common.inc
-include whitelist-runuser-common.inc
-include whitelist-usr-share-common.inc
-include whitelist-var-common.inc
+# These lines are needed to allow Firefox to open links
+noblacklist ${HOME}/.mozilla
+whitelist ${HOME}/.mozilla/firefox/profiles.ini
+read-only ${HOME}/.mozilla/firefox/profiles.ini
 
-apparmor
-caps.drop all
-netfilter
 no3d
-nodvd
-nogroups
-noinput
-nonewprivs
-noroot
-notv
-nou2f
-novideo
-protocol unix,inet,inet6,netlink
-seccomp !chroot
-# tracelog - breaks on Arch
-
-private-bin electron-mail
-private-cache
-private-dev
+
+private-bin basename,cut,electron-mail,gio,grep,mawk,readlink,sh,tr,xdg-open,xdg-settings
```

Comment:
How did you generated this list?

Reply:
That does not work. `--build` will only grab program for your environment. Either add all required programs or drop private-bin.